sdk/identity/identity/src/credentials/managedIdentityCredential/models.ts

```diff
@@ -14,6 +14,17 @@ export interface MSIConfiguration {
   resourceId?: string;
 }
 
+/**
+ * @internal
+ * Represents an access token with an expiration time.
```

Comment:
minor: I think this comment is too generic for the type name.

Reply:
updated